seedui-typescript/template/src/component/layout/PageLayout.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import {
+    DrawerLayout,
+    Drawer,
+    DrawerBody,
+    DrawerNavGroup,
+    DrawerHeader,
+    NavItem,
+    DrawerFooter,
+} from '@brightlayer-ui/react-components';
+import Dashboard from '@mui/icons-material/Dashboard';
+import LogoutIcon from '@mui/icons-material/Logout';
+import { GroupAdd } from '@mui/icons-material';
+import Menu from '@mui/icons-material/Menu';
+import * as PXBColors from '@brightlayer-ui/colors';
+import { Typography, useMediaQuery, useTheme } from '@mui/material';
+import { useDrawer } from '../../contexts/drawerContextProvider';
+import headerBackgroundImage from '../../assets/images/topology.png';
+import BrandLogo from '../../assets/images/seed_logo.png';
+import eatonlogo from '../../assets/images/eaton_logo.svg';
+import { LocalStorage } from '../../utils/local-storage';
+import { copyRightsStyle, rghtsReservedStyle } from './layoutStyle';
+
+const ROUTE_OBJ = [
+    {
+        path: '/invite',
+        name: 'inviteUser',
+    },
+    {
+        path: '/dashboard',
+        name: 'dashboard',
+    },
+];
+export const PageLayout: React.FC<React.PropsWithChildren> = (props) => {
+    const { drawerOpen, setDrawerOpen } = useDrawer();
+
+    const [active, setActive] = useState<string>('');
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+
+    const { pathname } = window.location;
+    const firstNavList: NavItem[] = [
+        {
+            itemID: 'dashboard',
+            title: 'Dashboard',
+            icon: <Dashboard />,
+            onClick: (): void => {
+                setActive('dashboard');
+                window.location.href = '/dashboard';
+            },
+            divider: false,
+        },
+        {
+            itemID: 'inviteUser',
+            title: 'Invite User',
+            icon: <GroupAdd />,
+            onClick: (): void => {
+                setActive('inviteUser');
```

Comment:
can you add something here to setDrawerOpen(false) for onclick so when nav item is selected the drawer closes and page is displayed...

onClick:
                                page.route !== undefined
                                    ? (): void => {
                                          handleNavigate(page.route);
                                          if (isMobile) setDrawerOpen(false);
                                      }

Reply:
also something like this for handleNav

const handleNavigate = useCallback(
        (id: string): void => {
            navigate(id);
            setSelected(id);
        },
        [navigate, setSelected]
    );